dev-python/doit/doit-0.32.0-r1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit bash-completion-r1 distutils-r1
+
+DESCRIPTION="Automation tool"
+HOMEPAGE="https://pydoit.org/ https://pypi.org/project/doit/"
+SRC_URI="mirror://pypi/${PN::1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-python/cloudpickle[${PYTHON_USEDEP}]
+	dev-python/pyinotify[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]"
+DEPEND="
+	test? (
+		${RDEPEND}
+		dev-python/mock[${PYTHON_USEDEP}]
+		dev-python/pyflakes[${PYTHON_USEDEP}]
+		<dev-python/pytest-5[${PYTHON_USEDEP}]
+	)"
+PDEPEND=">=dev-python/doit-py-0.4.0[${PYTHON_USEDEP}]"
+
+distutils_enable_tests pytest
```

Comment:
So python3.6 tests pass for me, while 3.7 tests fail. I'm not convinced that it's working properly...

Did they pass for you? I'm not seeing anything obvious.

```
tests/test_runner.py::TestJobTask::test_not_picklable_raises_InvalidTask /tmp/portage/dev-python/doit-0.32.0-r1/temp/environment: line 2927:   370 Aborted                 pytest -vv
 * ERROR: dev-python/doit-0.32.0-r1::testworld failed (test phase):
 *   Tests fail with python3.7
```

There is no FAILED to be seen anywhere.

Reply:
Strange enough, I don't have any test failures in 3.7. Seems https://github.com/pydoit/doit/issues/341 related. I'll try make patch on it.